src/cloudant/database.py

```diff
@@ -312,6 +312,8 @@ def get_view_result(self, ddoc_id, view_name, raw_result=False, **kwargs):
         :param bool reduce: True to use the reduce function, false otherwise.
         :param int skip: Skip this number of rows from the start.
             Not valid when used with :class:`~cloudant.result.Result` iteration.
+        :param bool stable: Whether or not the view results should be returned
+            from a "stable" set of shards.
         :param str stale: Allow the results from a stale view to be used. This
```

Comment:
Given that there is no specific way to deprecate this `stale` parameter we should consider either:
i) adding a comment here that `stable` and `update` are preferred
ii) delete this `:param` on the basis that it won't break anything while it is still listed in `RESULT_ARG_TYPES`

Reply:
How about i)

"Note that this parameter is deprecated and the appropriate combination of `stable` and `update` should be used instead."